websites/S/signed.host/metadata.json

```
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "service": "signed.host",
+  "altnames": ["bio.signed.host"],
+  "author": {
+    "id": "439084103690616844",
+    "name": "MalwarePad#0001"
+  },
+  "description": {
+    "en": "Signed.host is a private media, and bio page hosting service."
+  },
+  "url": ["signed.host", "bio.signed.host"],
```

Site uses `www` subdomain.
```suggestion
  "url": ["www.signed.host","signed.host", "bio.signed.host"],
```